pipeline/pipeline_def.json

```diff
@@ -1,5 +1,12 @@
 {
     "configs_def":[
-
+        {
+            "name": "engine_file",
+            "default_value": "${PIPELINE_SERVER_SETTINGS_MODELS_ROOT}"
+        },
+        {
+            "name":"class_map",
```

Comment:
What is class map?

Reply:
Right, not necessary here, removed class_map from pipeline class.